docs/cli.md

```diff
@@ -69,8 +69,10 @@ npx babel script.js --watch --out-file script-compiled.js
 
 ### Compile with Source Maps
 
+> **Note:** Since v7.19.3, if this parameter is not specified, `babel-cli` will follow the [configuration files](https://babeljs.io/docs/en/config-files).
```

Comment:
```suggestion
> **Note:** Since v7.19.3, if this parameter is not specified, `@babel/cli` will follow the [configuration files](https://babeljs.io/docs/en/config-files).
```

Reply:
Great find! I used to think this looked a little weird but couldn't find the reason.😂